reactor-core/src/main/java/reactor/core/publisher/UnicastManySinkNoBackpressure.java

```diff
@@ -153,16 +178,26 @@ public void emitError(Throwable error) {
 	@Override
 	public Emission tryEmitError(Throwable t) {
 		Objects.requireNonNull(t, "t");
-
-		switch (state) {
+		switch (this.state) {
 			case INITIAL:
-				// TODO different Emission?
-				return Emission.OK;
+				if (STATE.compareAndSet(this, State.INITIAL, State.TERMINATED_BEFORE_SUBSCRIPTION)) {
```

Comment:
consider extracting `this.state` into a variable and using it here in place of `State.INITIAL` to avoid future copy/paste errors